test/controllers/businessTradingName/CheckYourAnswersControllerSpec.scala

```diff
@@ -96,7 +96,7 @@ class CheckYourAnswersControllerSpec extends ControllerBaseSpec {
           lazy val result = {
             mockUpdateTradingName(vrn, UpdateTradingName(Some(testTradingName), None))(
               Future(Right(UpdateOrganisationDetailsSuccess("someFormBundle"))))
-            controller.updateTradingName()(requestWithTradingName)
+            controller.updateTradingName()(requestWithNewTradingName)
```

Comment:
This isn't a problem with your PR, but I notice we aren't unit testing the session values in this spec. Could you add a unit test for each place a session value is added? (there are 4 in total)